testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oid4vc/issuance/signing/OID4VCIssuerEndpointTest.java

```diff
@@ -430,6 +438,57 @@ public void testCredentialIssuance() throws Exception {
     }
 
 
+    // Tests the AuthZCode complete flow without scope from
+    // 1. Get authorization code without scope specified by wallet
+    // 2. Using the code to get access token 
+    // 3. Get the credential configuration id from issuer metadata at .wellKnown
+    // 4. With the access token, get the credential
+    @Test
+    public void testCredentialIssuanceWithAuthZCode() throws Exception {
+ 
+        // 1. Get authoriZation code without scope specified by wallet
+        // 2. Using the code to get accesstoken
+        String token = getBearerToken(oauth.openid(false).scope(null));
+ 
+        // 3. Get the credential configuration id from issuer metadata at .wellKnown
+        Client client = AdminClientUtil.createResteasyClient();
+        UriBuilder builder = UriBuilder.fromUri(OAuthClient.AUTH_SERVER_ROOT);
+        URI oid4vciDiscoveryUri = RealmsResource.wellKnownProviderUrl(builder).build("test", OID4VCIssuerWellKnownProviderFactory.PROVIDER_ID);
+        WebTarget oid4vciDiscoveryTarget = client.target(oid4vciDiscoveryUri);
+        Response discoveryResponse = oid4vciDiscoveryTarget.request().get();
+        CredentialIssuer oid4vciIssuerConfig = JsonSerialization.readValue(discoveryResponse.readEntity(String.class), CredentialIssuer.class);
+ 
+        assertEquals(200, discoveryResponse.getStatus());
+        assertEquals("https://localhost:8543/auth/realms/test", oid4vciIssuerConfig.getCredentialIssuer());
```

Comment:
If possible, could you avoid hard-coded  `https://localhost:8543/auth/realms/test` ?